src/explore-education-statistics-common/src/modules/table-tool/components/TableHeadersForm.tsx

```diff
@@ -26,6 +26,7 @@ interface Props {
 
 const TableHeadersForm = ({
   onSubmit,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   id = 'tableHeadersForm',
```

Comment:
Looks like `DataBlockSourceWizard.tsx` blows up if this prop is removed: 
```
TypeScript error in C:/Users/Hive/explore-education-statistics/src/explore-education-statistics-admin/src/pages/release/datablocks/components/DataBlockSourceWizard.tsx(79,11):
Type '{ initialValues: TableHeadersConfig; id: string; onSubmit: (nextTableHeaders: TableHeadersConfig) => Promise<void>; }' is not assignable to type 'IntrinsicAttributes & Props'.
  Property 'id' does not exist on type 'IntrinsicAttributes & Props'.  TS2322
```

This is because `DataBlockSourceWizard.tsx` is passing an id prop to TableHeadersForm: 

```
   <TableHeadersForm
          initialValues={currentTableHeaders}
          id="dataBlockSourceWizard-tableHeadersForm"
```

Reply:
To fix the lint properly, we need to actually use the `id` prop somewhere in this component. 

It looks like we should add this to the `Form` element.